src/main/java/org/openstreetmap/atlas/tags/filters/README.md

```diff
@@ -55,7 +55,7 @@ Anything without water=pond
 water->!pond,!lake
 ```
 
-Anything without water=pond or water=lake
+Anything without water=pond or without water=lake. That one is not very useful as it is always true because one of the two will always be true (The water tag can have only one value).
```

Comment:
It might be nice to have an example that shows how to properly do this with `water->!pond&water->!lake`.

Reply:
Let me do that!